gdal/doc/source/programs/gdaltindex.rst

```diff
@@ -89,14 +89,14 @@ tileindex.
 Examples
 --------
 
-- Produce a shapefile (``doq_index.shp``) with a record for every
-  image that the utility found in the ``doq`` folder. Each record holds
+- Produce a GeoPackage (``index.gpkg``) with a record for every
+  image that the utility found in the ``raster_files`` folder. Each record holds
   information that points to the location of the image and also a bounding rectangle
   shape showing the bounds of the image:
 
 ::
 
-    gdaltindex doq_index.shp doq/*.tif
+    gdaltindex index.gpkg raster_files/*.tif
```

Comment:
I would rather use an example `gdaltindex -f GPKG index.gpkg raster_files/*.tif`. GDAL does guess the output format from the extension but it does not hurt to be explicit.

Reply:
thanks for the comment. I applied your suggestion in 1343146